AnnoyancesFilter/sections/popups.txt

```diff
@@ -107,7 +107,8 @@ myflixertv.to###site-notice
 myflixertv.to#$##modalshare { display: none!important; }
 myflixertv.to#$#body { overflow: visible!important; }
 myflixertv.to#$#.modal-backdrop { display: none!important; }
-kiiw.icu,urbes.com.br,filmold.com##.modal
+litebit.eu#$#body { overflow: visible !important; }
+litebit.eumkiiw.icu,urbes.com.br,filmold.com##.modal
```

Comment:
No, only this way for `cookies_specific.txt`.
```
litebit.eu#$#body { overflow: visible !important; }
litebit.eu#$?#.modal > div.cookie-consent:upward(1) { display: none !important; }
```

Reply:
@wout-codes it seems those ext-css rules don't work as I expected. Better add `litebit.eu#%#//scriptlet('set-cookie', 'userAcceptedMarketingCookies', 'true')` instead of them.